api/client/transform/fortio.proto

```diff
@@ -41,6 +41,20 @@ message FortioResult {
   map<string, uint64> RetCodes = 11;
 
   string URL = 12;
+
+  string Version = 13;
```

Comment:
(optional) Unsure what role this proto plays. If it is "public" by any means we should improve the documentation of these fields. We can leave this to a follow-up PR.

Reply:
Followed up on the requested doc comments in 77a7eb267423cac7b15cde7a2a2c83e699177756 over at #265